app/cisc/commands.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import "gopkg.in/codegangsta/cli.v1"
+
+/*
+This holds the cli-commands so the main-file is less cluttered.
+*/
+
+var commandID, commandConfig, commandKeyvalue, commandSSH cli.Command
+
+func init() {
+	commandID = cli.Command{
+		Name:  "id",
+		Usage: "working on the identity",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Aliases:   []string{"cr"},
+				Usage:     "start a new identity",
+				ArgsUsage: "group [id-name]",
+				Action:    idCreate,
+			},
+			{
+				Name:      "connect",
+				Aliases:   []string{"co"},
+				Usage:     "connect to an existing identity",
+				ArgsUsage: "group id [id-name]",
+				Action:    idConnect,
+			},
+			{
+				Name:    "follow",
+				Aliases: []string{"f"},
+				Usage:   "follow an existing identity",
+				Action:  idFollow,
+			},
+			//{
+			//	Name:    "remove",
+			//	Aliases: []string{"rm"},
+			//	Usage:   "remove an identity",
+			//	Action:  idRemove,
+			//},
+			//{
+			//	Name:    "check",
+			//	Aliases: []string{"ch"},
+			//	Usage:   "check the health of the cothority",
+			//	Action:  idCheck,
+			//},
+		},
+	}
+	commandConfig = cli.Command{
+		Name:  "config",
+		Usage: "updating and voting on config",
+		Subcommands: []cli.Command{
+			{
+				Name:    "propose",
+				Aliases: []string{"l"},
+				Usage:   "propose the new config",
+				Action:  configPropose,
+			},
+			{
+				Name:    "update",
+				Aliases: []string{"u"},
+				Usage:   "fetch the latest config",
+				Action:  configUpdate,
+			},
+			{
+				Name:    "list",
+				Aliases: []string{"ls"},
+				Usage:   "list existing config and proposed",
+				Action:  configList,
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "p,propose",
+						Usage: "will also show proposed config",
+					},
+				},
+			},
+			{
+				Name:    "vote",
+				Aliases: []string{"v"},
+				Usage:   "vote on existing config",
+				Flags: []cli.Flag{
```

Comment:
Hmm - taking awayt the `reject`, have interactive if no `[yn]`-argument is given, and follow `[yn]` else? Does that make sense?